src/OpenTelemetry.AutoInstrumentation/Configurations/TracerSettings.cs

```diff
@@ -89,32 +89,47 @@ protected override void OnLoad(Configuration configuration)
         InstrumentationOptions = new InstrumentationOptions(configuration);
     }
 
-    private static TracesExporter ParseTracesExporter(Configuration configuration)
+    private static IReadOnlyList<TracesExporter> ParseTracesExporter(Configuration configuration)
     {
-        var tracesExporterEnvVar = configuration.GetString(ConfigurationKeys.Traces.Exporter)
-            ?? Constants.ConfigurationValues.Exporters.Otlp;
+        var tracesExporterEnvVar = configuration.GetString(ConfigurationKeys.Traces.Exporter);
 
-        switch (tracesExporterEnvVar)
+        if (string.IsNullOrWhiteSpace(tracesExporterEnvVar))
         {
-            case null:
-            case "":
-            case Constants.ConfigurationValues.Exporters.Otlp:
-                return TracesExporter.Otlp;
-            case Constants.ConfigurationValues.Exporters.Zipkin:
-                return TracesExporter.Zipkin;
-            case Constants.ConfigurationValues.None:
-                return TracesExporter.None;
-            default:
-                if (configuration.FailFast)
-                {
-                    var message = $"Traces exporter '{tracesExporterEnvVar}' is not supported.";
-                    Logger.Error(message);
-                    throw new NotSupportedException(message);
-                }
-
-                Logger.Error($"Traces exporter '{tracesExporterEnvVar}' is not supported. Defaulting to '{Constants.ConfigurationValues.Exporters.Otlp}'.");
-
-                return TracesExporter.Otlp;
+            return new List<TracesExporter> { TracesExporter.Otlp }.AsReadOnly();
         }
+
+        var exporters = new HashSet<TracesExporter>();
+
+        var exporterNames = tracesExporterEnvVar!.Split(',')
+                                                 .Select(e => e.Trim())
+                                                 .Where(e => !string.IsNullOrEmpty(e))
```

Comment:
This is different from how we are parsing propagators. We are not trimming the whitespaces and also we are not getting rid of empty entries.

See `SdkSettings.cs`.

The OpenTelemetry specification does not say anything about trimming whitespaces: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/configuration/sdk-environment-variables.md